packit/security.py

```diff
@@ -44,10 +44,10 @@ class CommitVerifier:
 
     def __init__(self, key_server: str = None) -> None:
         """
-        :param key_server: GPG key server to be used, defaults to keys.fedoraproject.org
+        :param key_server: GPG key server to be used, defaults to keys.openpgp.org
```

Comment:
Is this commit required? Maybe it could be squashed with the one introducing the list of keyservers.